spec/services/nomis/feature_spec.rb

```diff
@@ -107,4 +107,32 @@
       it { expect(described_class.slot_availability_enabled?(prison_name)).to eq(true) }
     end
   end
+
+  describe '.book_to_nomis_enabled?' do
+    context 'when the book to nomis flag is disabled' do
+      before do
+        switch_off :nomis_staff_book_to_nomis_enabled
+      end
+
+      it { expect(described_class.book_to_nomis_enabled?(anything)).to eq(false) }
```

Comment:
I find the use of predicate methods with variables not very readable when they take parameters. I think is clearer like it is now.